index.md

```diff
@@ -3,10 +3,11 @@ layout: default
 title: Semantic Versioning 2.0.0
 ---
 
+
 Semantic Versioning 2.0.0
 ==============================
 
-Summary
```

Comment:
Sorry if I wasn't clear before. I wanted these changes reverted too. I was hoping that this change would _only_ include the addition of named anchors. Changing the heading style here makes the headings all into `H1` elements which looks too large and gives these headings the same emphasis as the title.